.github/pull_request_template.md

```diff
@@ -0,0 +1,33 @@
+<!--
+Thanks for sending a pull request!  Here are some tips for you:
+  1. If this is your first time, please read our contributor guidelines: https://github.com/delta-incubator/delta-kernel-rs/blob/main/CONTRIBUTING.md
+  2. Ensure you have added or run the appropriate tests for your PR and run `cargo fmt`.
```

Comment:
can we add a quick one-liner for testing? Maybe something to the effect of "run `cargo t --all-features --all-targets` to get started testing"